core/process/aru.php

```diff
@@ -706,9 +706,21 @@
 		header('Content-Type: application/json');
 
 		echo $json;
-
 		break;
 
+	case 'pokemon_coordinates_area':
+		$json="";
+		$req 		 = "SELECT MAX(latitude) AS max_latitude, MIN(latitude) AS min_latitude, MAX(longitude) AS max_longitude, MIN(longitude) as min_longitude FROM pokemon";
```

Comment:
Hmm:

MariaDB [pokemonkiel]> SELECT MAX(latitude) AS max_latitude, MIN(latitude) AS min_latitude, MAX(longitude) AS max_longitude, MIN(longitude) as min_longitude FROM pokemon
+------------------+------------------+------------------+------------------+
| max_latitude     | min_latitude     | max_longitude    | min_longitude    |
+------------------+------------------+------------------+------------------+
| 54.5166280733962 | 53.9861774903964 | 10.6616758923218 | 9.48718048032647 |
+------------------+------------------+------------------+------------------+
1 row in set (0.06 sec)

But I can change it later, yes